value-object/src/main/java/com/iluwatar/value/object/App.java

```diff
@@ -0,0 +1,19 @@
+package com.iluwatar.value.object;
+
+/**
+ * App Class.
```

Comment:
Please describe the pattern and the example here. For reference, see other pattern examples.